cabal.project

```diff
@@ -14,10 +14,10 @@ repository cardano-haskell-packages
 -- We duplicate the hackage index-state first for haskell.nix, even though cabal ignores it.
 -- This must always match the hackage index-state on the next line.
 -- See https://github.com/input-output-hk/haskell.nix/issues/1869#issuecomment-1449272480
-index-state: 2023-03-05T00:00:00Z
+index-state: 2023-05-26T00:00:00Z
 index-state:
-  , hackage.haskell.org      2023-03-05T00:00:00Z
-  , cardano-haskell-packages 2023-04-03T17:08:55Z
+  , hackage.haskell.org      2023-05-26T00:00:00Z
+  , cardano-haskell-packages 2023-05-26T00:00:00Z
 
 -- with-compiler: ghc-8.10.7
```

Comment:
So, we're still using GHC 8.10.7?

Reply:
No, that's a vestigial comment and has no effect